src/main/java/com/artipie/management/api/Cookies.java

```diff
@@ -24,114 +24,63 @@
 package com.artipie.management.api;
 
 import com.artipie.http.auth.Authentication;
-import com.artipie.http.rq.RqHeaders;
-import com.jcabi.log.Logger;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.security.GeneralSecurityException;
-import java.security.KeyFactory;
-import java.security.spec.KeySpec;
-import java.security.spec.PKCS8EncodedKeySpec;
-import java.util.HashMap;
-import java.util.Locale;
 import java.util.Map;
 import java.util.Optional;
-import javax.crypto.Cipher;
-import org.apache.commons.codec.DecoderException;
-import org.apache.commons.codec.binary.Hex;
 
 /**
- * API request cookies.
- *
- * @since 0.1
+ * Cookies.
+ * @since 0.2
  */
-public final class Cookies {
-
-    /**
-     * Headers.
-     */
-    private final Iterable<Map.Entry<String, String>> headers;
-
-    /**
-     * Ctor.
-     *
-     * @param headers Headers.
-     */
-    public Cookies(final Iterable<Map.Entry<String, String>> headers) {
-        this.headers = headers;
-    }
+public interface Cookies {
```

Comment:
@olenagerasimova this interface looks surprisingly similar to `AuthScheme` that was added to HTTP module. Maybe it makes sense to use it here?

Reply:
@olegmoz thanks, I think it's a good idea. But I'll need `AuthScheme.Fake` implementation for the test. Let me submit corresponding PR in `http` and use `AuthScheme` here